src/main/java/org/opentripplanner/transit/raptor/rangeraptor/multicriteria/Boarding.java

```diff
@@ -10,6 +10,34 @@
  * while possessing the stops in the pattern. This class keep the needed state for these
  * boardings to avoid recalculating each value more than once and to be able put then in a
  * {@link org.opentripplanner.transit.raptor.util.paretoset.ParetoSet}.
+ * <p>
+ * A boarding represent one path from the {@code origin} to a trip is boarded(onboard of a vehicle).
+ * It represent the STATE of riding a trip, having boarded at a given stop, but not yet alighted.
+ * <p>
+ * We do not do this the same way as described in the original Raptor paper. The original McRaptor
+ * algorithm keep a bag of stop-arrivals while traversing the pattern. We keep a "bag"
```

Comment:
stop-arrival (labels)

Reply:
I have updated the documentation as best I could. I do not think "Labels" is the correct word here, so I did not use it. Alternative for `StopArrival` could be `StopReached`, and for `Boarding` could be `PatternBoarded` or `TripBoarded`.